assets/csidriveroperators/aws-ebs/hypershift/guest/generated/rbac.authorization.k8s.io_v1_role_aws-ebs-csi-driver-operator-role.yaml

```diff
@@ -22,19 +22,11 @@ rules:
   - update
   - patch
   - delete
-- apiGroups:
-  - ''
-  resources:
-  - namespaces
-  verbs:
-  - get
 - apiGroups:
   - apps
   resources:
   - deployments
-  - daemonsets
```

Comment:
IMO the operator will need to create / manage driver DaemonSet in the guest cluster
(right now, the operator uses some admin kubeconfig, but we should move to stricter role eventually)